scripts/ccr-custom-changelog/release.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const conventionalChangelog = require('conventional-changelog-conventionalcommits');
+const Q = require('q');
+
+const config = {
+	types: [
+		{ type: "feat", section: "Features" },
+		{ type: "fix", section: "Bug Fixes" },
+		{ type: "perf", section: "Performance Improvements", hidden: false },
+		{ type: "test", section: "Test Suite", hidden: false },
+		{ type: "build", section: "Build System", hidden: false },
+		{ type: "docs", section: "Documentation", hidden: false },
+		{ type: "style", section: "Other", hidden: false },
+		{ type: "refactor", section: "Other", hidden: false },
```

Comment:
I've been using `ref` as a prefix in commits for refactors. Is this suggested addition appropriate?
```suggestion
		{ type: "ref", section: "Other", hidden: false },
		{ type: "refactor", section: "Other", hidden: false },
```

Reply:
We could change it but that would also mean adding it to the allowed types in the linter. Also I think ref is more ambiguous than refactor. (Reference, refactor, reformat, refine)